Products/ZenModel/ZenossInfo.py

```diff
@@ -279,6 +281,7 @@ def getZopeVersion(self):
         major, minor, micro, status, release = version.getZopeVersion()
         return Version(name, major, minor, micro)
 
+    @FunctionCache('getZenossRevision')
```

Comment:
It sounds like we should remove this method at all. @jplouis please confirm it and I'll create an another pull request